src/System.Net.Requests/ref/System.Net.Requests.cs

```diff
@@ -46,198 +42,245 @@ public class FileWebRequest : WebRequest, System.Runtime.Serialization.ISerializ
         public long ContentOffset { get { throw null; } set { } }
         public override string ContentType { get { throw null; } set { } }
         public override System.Net.ICredentials Credentials { get { throw null; } set { } }
+        public static new System.Net.Cache.RequestCachePolicy DefaultCachePolicy { get { throw null; } set { } }
+        public bool EnableSsl { get { throw null; } set { } }
         public override System.Net.WebHeaderCollection Headers { get { throw null; } set { } }
+        public bool KeepAlive { get { throw null; } set { } }
         public override string Method { get { throw null; } set { } }
         public override bool PreAuthenticate { get { throw null; } set { } }
         public override System.Net.IWebProxy Proxy { get { throw null; } set { } }
+        public int ReadWriteTimeout { get { throw null; } set { } }
         public string RenameTo { get { throw null; } set { } }
+        public override System.Uri RequestUri { get { throw null; } }
+        public System.Net.ServicePoint ServicePoint { get { throw null; } }
         public override int Timeout { get { throw null; } set { } }
-        public bool EnableSsl { get { throw null; } set { } }
-        public bool UsePassive { get { throw null; } set { } }
         public bool UseBinary { get { throw null; } set { } }
-        public bool KeepAlive { get { throw null; } set { } }
-        public int ReadWriteTimeout { get { throw null; } set { } }
-        public System.Net.ServicePoint ServicePoint { get { throw null; } }
-        public static new System.Net.Cache.RequestCachePolicy DefaultCachePolicy { get { throw null; } set { } }
-        public override System.Uri RequestUri { get { throw null; } }
         public override bool UseDefaultCredentials { get { throw null; } set { } }
-        public override void Abort() { throw null; }
+        public bool UsePassive { get { throw null; } set { } }
+        public override void Abort() { }
         public override System.IAsyncResult BeginGetRequestStream(System.AsyncCallback callback, object state) { throw null; }
         public override System.IAsyncResult BeginGetResponse(System.AsyncCallback callback, object state) { throw null; }
         public override System.IO.Stream EndGetRequestStream(System.IAsyncResult asyncResult) { throw null; }
         public override System.Net.WebResponse EndGetResponse(System.IAsyncResult asyncResult) { throw null; }
         public override System.IO.Stream GetRequestStream() { throw null; }
         public override System.Net.WebResponse GetResponse() { throw null; }
     }
-    public class FtpWebResponse : WebResponse, IDisposable
+    public partial class FtpWebResponse : System.Net.WebResponse, System.IDisposable
     {
         internal FtpWebResponse() { }
-        public override long ContentLength { get { throw null; } }
-        public override string ContentType { get { throw null; } }
```

Comment:
Removed. Is that fine?

Reply:
"public override long ContentLength" needs to stay.  That is the current public API in .NET Framework as well.

However, .NET Framework doesn't have a "public overrride string ContentType" for this class.  But since FtpWebResponse is derived from WebResponse, there is a 'WebResponse.ContentType' property already.

Reply:
> "public override long ContentLength" needs to stay

No, since the type is `sealed` and the property is not declared `abstract` there's no requirement that the override be listed in the ref.cs (the declaration of overrides is only required to make `base.Member` bind correctly; and to tell the compiler that all abstracts have been fulfilled).